ballerina/Package.md

```diff
@@ -6,7 +6,7 @@ gRPC is an inter-process communication technology that allows you to connect, in
 
 When you develop a gRPC application, the first thing you do is define a service definition using Protocol Buffers.
 
-### Protocol Buffers
+### Protocol buffers
```

Comment:
```suggestion
### Protocol Buffers
```
Industrial terms should be kept as per the standard.

Reply:
`Protocol buffers` is correct - https://developers.google.com/protocol-buffers